src/applications/edu-benefits/0994/containers/ConfirmationPage.jsx

```diff
@@ -60,6 +65,21 @@ class ConfirmationPage extends React.Component {
           <br />
           <i>Please print this page for your records.</i>
         </p>
+        {!appliedForVaEducationBenefits && (
+          <AlertBox
+            isVisible
```

Comment:
You could pass `!appliedForVaEducationBenefits to this `isVisible` prop, instead of putting `AlertBox` in this conditiona.

Reply:
That is definitely a cleaner approach. Thanks for pointing it out.